docs/guides/alert-manager-configuration.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "Configuring Notification using Cortex AlertManager"
```

Comment:
The alertmanager is spelled "Alertmanager" in Prometheus ([see here](https://prometheus.io/docs/alerting/latest/alertmanager/)) so I would change "AlertManger" to "Alertmanager" across this PR to keep it consistent with Prometheus.